prov/gni/src/gnix_eq.c

```diff
@@ -65,14 +97,24 @@ static int gnix_verify_eq_attr(struct fi_eq_attr *attr)
 	 */
 	switch (attr->wait_obj) {
 	case FI_WAIT_NONE:
-	case FI_WAIT_UNSPEC:
 		break;
 	case FI_WAIT_SET:
+		if (!attr->wait_set) {
+			GNIX_WARN(FI_LOG_EQ,
+				  "FI_WAIT_SET is set, but wait_set field doesn't reference a wait object.\n");
```

Comment:
What's the issue? You mean the long string? The linux style guide says that strings should not be broken up because it breaks the grep-ability. So if a string is over 80 chars, but it's the only thing on the line then it's acceptable.